Common/src/Pollster/CreateLargeTaskProcessor.cs

```diff
@@ -0,0 +1,188 @@
+                                                                                         taskRequestsChannel_.Reader.ReadAllAsync(cancellationToken),
+                                                                                         cancellationToken).ConfigureAwait(false);
+
+                                      await asyncPipe_.WriteAsync(new ProcessRequest
+                                                                  {
+                                                                    CreateTask = new ProcessRequest.Types.CreateTask
+                                                                                 {
+                                                                                   Reply = new CreateTaskReply
+                                                                                           {
+                                                                                             Successfull = new Empty(),
+                                                                                           },
+                                                                                   ReplyId = requestId_,
+                                                                                 },
+                                                                  })
+                                                      .ConfigureAwait(false);
+                                   },
+                                   cancellationToken);
+        break;
+      case ProcessReply.Types.CreateLargeTaskRequest.TypeOneofCase.InitTask:
+
+        switch (processReply.CreateLargeTask.InitTask.TypeCase)
+        {
+          case InitTaskRequest.TypeOneofCase.Header:
+            fsm_.AddHeader();
+            payloadsChannel_ = Channel.CreateUnbounded<ReadOnlyMemory<byte>>(new UnboundedChannelOptions
+                                                                             {
+                                                                               SingleWriter = true,
+                                                                               SingleReader = true,
+                                                                             });
+
+            await taskRequestsChannel_.Writer.WriteAsync(new TaskRequest(processReply.CreateLargeTask.InitTask.Header.Id,
+                                                                         processReply.CreateLargeTask.InitTask.Header.ExpectedOutputKeys,
+                                                                         processReply.CreateLargeTask.InitTask.Header.DataDependencies,
+                                                                         payloadsChannel_.Reader.ReadAllAsync(cancellationToken)),
+                                                         cancellationToken)
+                                      .ConfigureAwait(false);
+            
+
+            break;
```

Comment:
Syntax question: the `break`keyword is a need?

Reply:
Yes, there are 2 nested switch cases